doc/nrf/app_dev/programming.rst

```diff
@@ -21,6 +21,41 @@ To program the :ref:`output build files <app_build_output_files>` to your device
 The flash command programs all cores by default, both in the |nRFVSC| and on the command line.
 If you want to program only one selected core, use ``west flash`` on the command line and :ref:`specify the domain <zephyr:west-multi-domain-flashing>`.
 
+.. _programming_selecting_runner:
+
+Selecting west runner
+*********************
+
+Starting with the |NCS| v3.0.0, all Nordic Semiconductor :ref:`boards <app_boards>` are using the `nRF Util`_ as the default runner for the ``west flash`` and ``west debug`` command.
+This change ensures that the flashing process is consistent across all boards.
+See the :ref:`build system section in the v3.0.0 migration guide <migration_3.0_recommended>` for more information.
+
+Runners are Zephyr-specific Python classes that wrap Zephyr's :ref:`flash and debug host tools <zephyr:flash-debug-host-tools>` and integrate them with west and the Zephyr build system to support ``west flash`` and related commands.
```

Comment:
You could mention that the runners supported on a board are determined by the `board.cmake` file in the board folder, see for example [this one](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf52840dk/board.cmake).  In particular, it's the `include(${ZEPHYR_BASE}/boards/common/<runner>.board.cmake)` lines that enable them. The order matters, the first of those lines becomes the default runner.